src/framework/global/types/retval.h

```diff
@@ -31,9 +31,12 @@ template<typename T>
 struct RetVal {
     Ret ret;
     T val;
+
     RetVal() = default;
     RetVal(const Ret& r)
         : ret(r) {}
+    RetVal(const T& v)
```

Comment:
Not sure about this change. Such constructors very often cause various kinds of problems because they do things implicitly

Reply:
Yeah, I doubted about it too. Probably I should just be not too lazy to write `RetVal<QString>::make_ok()` :)